environment.yml

```diff
@@ -0,0 +1,8 @@
+name: myenv
+channels:
+  - defaults
+dependencies:
+  - python=3.6
+  - codecov
+  - pytest-cov
```

Comment:
Add documentation dependencies as well.

Reply:
See, `docs/requirements.txt` inside `pydatastructs` root.

Reply:
conda does not have sphinx_readable_theme package in it what to do

Reply:
I think you have already found the solution. Does that work?